extensions/mssql/src/hdfs/ui/hdfsManageAccessDialog.ts

```diff
@@ -86,170 +83,250 @@ export class ManageAccessDialog {
+		const pathLabel = this.modelBuilder.text()
+			.withProperties<azdata.TextComponentProperties>({
+				value: this.hdfsPath,
+				title: this.hdfsPath,
+				height: locationLabelHeight,
+				CSSStyles: { 'user-select': 'text', 'overflow': 'hidden', 'text-overflow': 'ellipsis', ...cssStyles.titleCss }
+			}).component();
+
+		locationContainer.addItem(locationLabel,
+			{
+				flex: '0 0 auto',
+				CSSStyles: { 'margin-bottom': '5px' }
+			});
+		locationContainer.addItem(pathLabel,
+			{
+				flex: '1 1 auto',
+				CSSStyles: { 'border': '1px solid #ccc', 'padding': '5px', 'margin-left': '10px', 'min-height': `${locationLabelHeight}px` }
+			});
 
-				// =====================
-				// = Permissions Title =
-				// =====================
-				const permissionsTitle = modelView.modelBuilder.text()
-					.withProperties<azdata.TextComponentProperties>({ value: loc.permissionsHeader })
-					.component();
-				contentContainer.addItem(permissionsTitle, { CSSStyles: { 'margin-top': '15px', ...cssStyles.titleCss } });
+		contentContainer.addItem(locationContainer, { flex: '0 0 auto', CSSStyles: { 'margin-top': '20px' } });
 
-				// =============================
-				// = POSIX permissions section =
-				// =============================
+		// =====================
+		// = Permissions Title =
+		// =====================
+		const permissionsTitle = this.modelBuilder.text()
+			.withProperties<azdata.TextComponentProperties>({ value: loc.permissionsHeader })
+			.component();
+		contentContainer.addItem(permissionsTitle, { CSSStyles: { 'margin-top': '15px', ...cssStyles.titleCss } });
 
-				const posixPermissionsContainer = this.createPermissionsHeaderRow(modelView.modelBuilder, '', /*includeInherit*/false, /*includeStickyAndInherit*/true);
-				contentContainer.addItem(posixPermissionsContainer, { CSSStyles: { ...cssStyles.tableHeaderLayoutCss } });
+		// ====================
+		// = Inherit Defaults =
+		// ====================
 
-				// Empty initially - this is going to eventually be populated with the owner/owning/other group permissions
-				this.posixPermissionsContainer = modelView.modelBuilder.flexContainer().withLayout({ flexFlow: 'column' }).component();
-				contentContainer.addItem(this.posixPermissionsContainer, { flex: '0 0 auto', CSSStyles: { 'margin-bottom': '20px' } });
+		// Defaults are only settable for directories
+		if (this.hdfsModel.fileStatus.type === HdfsFileType.Directory) {
+			contentContainer.addItem(this.createInheritDefaultsCheckbox());
+		}
 
-				// ===========================
-				// = Add User Or Group Input =
-				// ===========================
+		// ==========
+		// = Sticky =
+		// ==========
+		this.stickyCheckbox = this.modelBuilder.checkBox()
+			.withProperties<azdata.CheckBoxProperties>({
+				width: checkboxSize,
+				height: checkboxSize,
+				checked: permissionStatus.stickyBit,
+				label: loc.stickyLabel
+			}).component();
+		this.stickyCheckbox.onChanged(() => {
```

Comment:
I noticed we are not handling the event disposables in this dialog, can we add these to an array and dispose them when the dialog is getting closed?

Reply:
I'll see about doing this as a follow up task - not going to add extra stuff to this PR at this point